app/services/analytics_events.rb

```diff
@@ -763,21 +809,35 @@ def idv_cancellation_visited(
     )
   end
 
-  def idv_capture_troubleshooting_dismissed(**_extra)
+  # @param [Boolean] acuant_sdk_upgrade_a_b_testing_enabled
+  # @param [String] acuant_version
+  # @param [String] flow_path whether the user is in the hybrid or standard flow
+  # @param [String] use_alternate_sdk
+  def idv_capture_troubleshooting_dismissed(
+    acuant_sdk_upgrade_a_b_testing_enabled:,
+    acuant_version:,
+    flow_path:,
+    use_alternate_sdk:,
+    **_extra
+  )
     track_event(
       'Frontend: IdV: Capture troubleshooting dismissed',
+      acuant_sdk_upgrade_a_b_testing_enabled: acuant_sdk_upgrade_a_b_testing_enabled,
+      acuant_version: acuant_version,
+      flow_path: flow_path,
+      use_alternate_sdk: use_alternate_sdk,
     )
   end
 
   def idv_clicked_sp_link_on_ready_to_verify_page(**_extra)
     track_event(
-      'Frontend: IdV: user clicked sp link on ready to verify page',
+      'IdV: user clicked sp link on ready to verify page',
     )
   end
 
   def idv_clicked_what_to_bring_link_on_ready_to_verify_page(**_extra)
     track_event(
-      'Frontend: IdV: user clicked what to bring link on ready to verify page',
+      'IdV: user clicked what to bring link on ready to verify page',
```

Comment:
Just want to flag that these renames will now fail build as of #9539. Renaming an event is a good opportunity to adopt the new naming scheme.

These should be `:idv_clicked_sp_link_on_ready_to_verify_page` and `:idv_clicked_what_to_bring_link_on_ready_to_verify_page` now if we're okay to do a rename.

Reply:
thanks Andrew .. removed this unused method as we discussed 👍🏿

Reply:
I _think_ this one was translated incorrectly by me to begin with and should not have included the Frontend prefix?

Reply:
@mitchellhenke I think we didn't need the method you added, it was already wired up to an existing analytics method that doesn't have the prefix. The method you added was effectively dead code.

https://github.com/18F/identity-idp/blob/5209813da52356d011ddfedcfd61f4b442b912ee/app/controllers/frontend_log_controller.rb#L39

https://github.com/18F/identity-idp/blob/5209813da52356d011ddfedcfd61f4b442b912ee/app/services/analytics_events.rb#L1826-L1832

Reply:
ahhhh, thanks